start_dev_services.sh

```
@@ -0,0 +1,74 @@
+  -e KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1 \
+  confluentinc/cp-kafka:7.9.0
+
+echo "Waiting for services to become healthy..."
+
+check_port() {
+  local port=$1
+  local name=$2
+  local retries=30
+  local wait=2
+  echo -n "Waiting for $name on port $port..."
+  while ! nc -z 127.0.0.1 $port >/dev/null 2>&1; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)